lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -648,11 +750,16 @@ func deleteSubscriber(networkID, key string) error {
 	return nil
 }
 
-func loadAllStatesBySID(networkID string) (map[string]state_types.StatesByID, error) {
-	states, err := state.SearchStates(networkID, subscriberStateTypes, nil, nil, serdes.State)
+func loadAllStatesBySIDForIMSIs(networkID string, imsis []string) (map[string]state_types.StatesByID, error) {
+	imsiKeyStates, err := state.SearchStates(networkID, subscriberStateTypesKeyedByIMSI, imsis, nil, serdes.State)
 	if err != nil {
 		return nil, err
 	}
+	imsiCompositeKeyStates, err := state.SearchStates(networkID, subscriberStateTypesKeyedByCompositeKey, nil, nil, serdes.State)
```

Comment:
This is unfortunate that we're still having to read in all subscriber state of a particular type... not the end of the world I guess, but we really need to deal with the whole "IMSI as a prefix of the key" thing, it feels hacky and results in suboptimalities like this